op-batcher/batcher/driver.go

```diff
@@ -268,6 +268,7 @@ func (l *BatchSubmitter) loop() {
 
 	receiptsCh := make(chan txmgr.TxReceipt[txID])
 	queue := txmgr.NewQueue[txID](l.killCtx, l.Txmgr, l.Config.MaxPendingTransactions)
+	var daWaitGroup sync.WaitGroup
```

Comment:
I'd create a similar interface to `txmgr.Queue` to hide impl details on how you're doing this

Reply:
Similarly, we should have a config option like `l.Config.MaxPendingTransactions` but for DA.